src/components/SidePanels/OutlinerPanel/OutlinerTree.tsx

```diff
@@ -379,7 +379,7 @@ const RegionControls: FC<RegionControlsProps> = observer(({
   const hidden = useMemo(() => {
     if (type?.includes('region') || type?.includes('range')) {
       return entity.hidden;
-    } else if ((!type || type.includes('label')) && regions) {
+    } else if ((!type || type.includes('label')) && regions || type?.includes('tool') && regions) {
```

Comment:
its a little ambiguous but I think this is what is intended 
```suggestion
    } else if (((!type || type.includes('label')) && regions || (type?.includes('tool') && regions)) {
```
which means that this line can be refactored to be a little easier to read
maybe something like 
```suggestion
    } else if ((!type || type?.includes('label') || type?.includes('tool')) && regions) {
```